finetune/base_models/glove/model.py

```diff
@@ -0,0 +1,43 @@
+import os
+from urllib.parse import urljoin
+
+from finetune.base_models import SourceModel
+from finetune.base_models.glove.encoder import GloveEncoder
+from finetune.base_models.glove.featurizer import glove_featurizer
+from finetune.util.download import GLOVE_BASE_URL, FINETUNE_BASE_FOLDER
+
+class GloveModel(SourceModel):
+    is_bidirectional = False
+    encoder = GloveEncoder
+    featurizer = glove_featurizer
+    settings = {
+        'batch_size': 128,
+        'n_embed_featurizer': 300,
+        'n_epochs': 100,
+        'keep_best_model': True,
+        'early_stopping_steps': 100,
+        'val_size': "auto",
+        'n_layer': 1,
```

Comment:
Honestly don't think I have to set half these params because they simply aren't used.